packages/jest-circus/src/run.ts

```diff
@@ -29,6 +29,8 @@ const run = async (): Promise<Circus.RunResult> => {
   );
 };
 
+const sleep = ms => new Promise(resolve => setTimeout(resolve, ms));
```

Comment:
Does this work if time is mocked? This code runs inside the user's sandbox

Reply:
I think it would work if we mock the `setTimeOut` bit.

Reply:
It might be better to add support via our babel plugin so our own timeouts (and intervals) are not affected by user code: https://github.com/facebook/jest/blob/3d36b608992fc56e0aa7295429bb80dbc8774ea9/scripts/babel-plugin-jest-native-globals.js

Reply:
I'll have a look on this and get back 😁